apisix/core/ctx.lua

```diff
@@ -207,6 +207,7 @@ do
         balancer_ip = true,
         balancer_port = true,
         consumer_name = true,
+        consumer_group_id = true,
```

Comment:
Why should we expose it as a variable? It seems we also use `ctx.consumer.group_id` in some places.

Reply:
We would use this variable in plugins:

For example:
```
curl http://127.0.0.1:9080/apisix/admin/consumer_groups/company_a -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
{
    "plugins": {
        "quota": {
            "rejected_code": 503,
            "minute": 2,
            "group": "$consumer_group_id"
        }
    }
}'
```